kubernetes-azure-go/go.mod

```
@@ -41,7 +41,7 @@ require (
 	github.com/opentracing/opentracing-go v1.1.0 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pkg/term v1.1.0 // indirect
-	github.com/pulumi/pulumi-azure-native-sdk v2.0.0 // indirect
+	github.com/pulumi/pulumi-azure-native-sdk v2.1.1 // indirect
```

`kubernetes-azure-go` complains:

```
go: errors parsing go.mod:
/Users/cnunciato/Projects/dev/prs/templates/kubernetes-azure-go/go.mod:44:2: require github.com/pulumi/pulumi-azure-native-sdk: version "v2.1.1" invalid: should be v0 or v1, not v2
error: installing dependencies failed; rerun manually to try again, then run `pulumi up` to perform an initial deployment: `go mod tidy` failed to install dependencies: exit status 1
```

It looks like this actually broke with https://github.com/pulumi/templates/pull/612/files#diff-95d2508a8ceb8a4cf1fd4e4dc299f16417a86f17e603eeb85cc3f381843d88edR44

I think it should be this:

```suggestion
	github.com/pulumi/pulumi-azure-native-sdk/v2 v2.1.1 // indirect
```

---

I'll go ahead and commit this and include it with this PR.

---

Thank you for catching that!